src/components/gamespec/Board.js

```diff
@@ -71,9 +80,23 @@ class Board extends React.Component {
     this.props.setItems(items);
   };
 
+  handleClickOn = index => {
+    let items = this.props.getItems();
+    let item = items[index];
+    if (item !== null && item.element.elementKind === 'toggable') {
```

Comment:
Instead of checking specifically equal to null, better check would be to check `if item &&`